### ISSUE TITLE
Differentiable Control Barrier Functions for Vision-based End-to-End Autonomous Driving. (arXiv:2203.02401v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#16425]
Differentiable Control Barrier Functions for Vision-based End-to-End Autonomous Driving. (arXiv:2203.02401v1 [cs.RO])<br>
https://ift.tt/RFIjdVG<br>
<p>Guaranteeing safety of perception-based learning systems is challenging due to the absence of ground-truth state information unlike in state-aware control scenarios. In this paper, we introduce a safety guaranteed learning framework for vision-based end-to-end autonomous driving. To this end, we design a learning system equipped with differentiable control barrier functions (dCBFs) that is trained end-to-end by gradient descent. Our models are composed of conventional neural network architectures and dCBFs. They are interpretable at scale, achieve great test performance under limited training data, and are safety guaranteed in a series of autonomous driving scenarios such as lane keeping and obstacle avoidance. We evaluated our framework in a sim-to-real environment, and tested on a real autonomous car, achieving safe lane following and obstacle avoidance via Augmented Reality (AR) and real parked vehicles.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
